docs/api-report/uix-core.api.md

```diff
@@ -144,6 +145,7 @@ export type GuestEmitter = GuestConnection & Emits<GuestConnectionEvents>;
 
 // @internal
 export interface HostConnection<T = unknown> {
+    getConfiguration(): Record<string, unknown> | undefined;
```

Comment:
Should we consider `null` instead of `undefined`?

Reply:
Do you want to capture the case where the extension is not configured yet? If not, I would love to see only `Record<string, unknown>` type. The record could be empty

Reply:
@kokoc , Not sure if I got your question right ...
Some extensions might not be configurable at all if developers didn't implement such functionality, or have some meaningful default values for all configurable parameters, so it is a valid case to not having a configuration value, that is why I marked it as `optional` (`undefined`).

Reply:
@lenaorobei , could you please describe benefits of setting `null` instead of `undefined`? `configuration` is an optional field, and it is totally fine not having it, see my answer above